test/SpokePoolClient.ValidateFill.ts

```diff
@@ -122,6 +122,16 @@ describe("SpokePoolClient: Fill Validation", function () {
     expect(filled.eq(deposit.amount)).is.true;
   });
 
+  it.skip("Tracks v3 fill status", async function () {
```

Comment:
Flagging a todo - this should be resolved in this PR.

Reply:
Update - this can't be resolved within this PR because this test queries the SpokePool contract directly via `buildFill()` and `relayFilledAmount()`. We need to bump contracts-v2 before we're able to do that.